index.html

```diff
@@ -12,12 +12,71 @@ <h2>Groceries list</h2>
     <h3>Lorem ipsum dolor sit amet consectetur adipisicing elit. Reiciendis voluptatem tempora quisquam maxime libero ipsam rerum sit, ducimus laboriosam saepe veniam neque adipisci praesentium labore. Esse cupiditate perspiciatis ea at.
     Totam, nihil! Omnis eligendi quis enim quidem maxime vitae quam ipsum accusantium esse eos sequi tenetur id repellat quisquam dicta asperiores repellendus, at suscipit sunt ex! Molestias, architecto dignissimos? Sapiente?</h3>
     <h4>Lorem, ipsum dolor sit amet consectetur adipisicing elit. Velit, similique accusamus. Cum tempore corporis quibusdam perspiciatis magni non illo commodi porro, voluptatum, dolores minima voluptate recusandae est sint veritatis ullam?</h4>
-    <h2>Items to buy:</h2>
+    <h3>Items to buy:</h3>
     <ol>
         <li>Bread</li>
         <li>Salt</li>
         <li>Sugar</li>
         <li>Paper</li>
         <li>Pork</li>
+<h2>Registration form</h2>
+
+<label>
+          <p>Address info</p>
+          <hr>
+      
+          <label for="City"><b>City</b></label>
+          <input type="text" placeholder="Enter City" name="City" id="City" required>
+      
+          <label for="Street Name"><b>Street Name</b></label>
+          <input type="Street Name" placeholder="Enter Street Name" name="Street Name" id="Street Name" required>
+      
+          <label for="Street number"><b>Street number</b></label>
+          <input type="Street number" placeholder="Street number" name="Street number" id="Street number" required>
+          <hr>
+          <p>Personal info</p>
+          <hr>
+      
+          <label for="First name"><b>City</b></label>
+          <input type="text" placeholder="First name" name="First name" id="First name" required>
+      
+          <label for="Last name"><b>Street Name</b></label>
+          <input type="text" placeholder="Last name" name="Last name" id="Last name" required>
+          <hr>
+    <textarea id="story" name="story" rows="5" cols="33">Other info</textarea>
```

Comment:
The controls like `input` and `textarea` can't have text inside them. Try to use the `placeholder` attribute instead